vignettes/Status.Rmd

```diff
@@ -0,0 +1,57 @@
+---
+title: "dataRetrieval Status"
+date: Sys.Date()
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    fig_caption: yes
+    fig_height: 7
+    fig_width: 7
+vignette: >
+  %\VignetteIndexEntry{dataRetrieval Status}
+  \usepackage[utf8]{inputenc}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+This page will be updated frequently with information about the status of dataRetrieval services.
+
+# Current Status
+
+## readNWISqw
+
+Functions normally, includes new data. This function is deprecated, so consider switching to `readWQPqw`
+
+## WQP
+
+Functioning normally, includes new USGS data. See below for upcoming information.
+
+# Overview of changes
+
+## NWIS qw services
+
+Discrete water samples data are undergoing modernization, and NWIS services will no longer being updated with the latest data starting in February 2024, with a full decommission expected 6 months later. Learn more about the upcoming change and where to find the new samples data in our [blog](https://waterdata.usgs.gov/blog/changes-to-sample-data/).
```

Comment:
misspelling: will --> with

Reply:
sorry, what "will" needs to change? I'm not seeing it

Reply:
sorry that wasn't clear: on reading again, this makes more sense: 'and NWIS services will no longer be updated...', so change being-->be instead.